databases/hiredis/Portfile

```diff
@@ -4,38 +4,28 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
 
```

Comment:
@reneeotten Could you add `legacysupport` to save from a need of subsequent fixes? Needed for `clock_gettime`: https://github.com/macos-powerpc/powerpc-ports/commit/316d779e1752d097dc7c1586cb9a13426477efc0